src/Parallel/Invoke.hpp

```diff
@@ -18,7 +45,46 @@ namespace Parallel {
  * \note The reason there are two separate functions is because Charm++ does not
  * allow defaulted arguments for group and nodegroup chares.
  */
-template <typename ReceiveTag, typename Proxy, typename ReceiveDataType>
+template <
+    typename ReceiveTag, typename Proxy, typename ReceiveDataType,
+    Requires<detail::has_ckLocal_method<std::decay_t<Proxy>>::value> = nullptr>
+void receive_data(Proxy&& proxy, typename ReceiveTag::temporal_id temporal_id,
+                  ReceiveDataType&& receive_data,
+                  const bool enable_if_disabled) noexcept {
```

Comment:
It looks like the functions in ActionTesting just default the `enable_if_disabled`. Are there other parts of the code that will break if you default here as well rather than having the two overloads?

Reply:
Yes, there is a bug in Charm++ 6.8 that causes this issue. See https://github.com/sxs-collaboration/spectre/blob/8aad31827187b8708eb202c092a02ca3438ed8a8/cmake/SetupCharmAlgorithm.py#L57